azkaban-exec-server/src/main/java/azkaban/execapp/event/JobCallbackManager.java

```diff
@@ -144,17 +148,13 @@ private void processJobCallOnFinish(final Event event) {
         JobCallbackUtil.buildJobContextInfoMap(event, this.azkabanHostName);
 
     JobCallbackStatusEnum jobCallBackStatusEnum = null;
-    final Logger jobLogger = jobRunner.getLogger();
-
+    final Logger jobLogger = this.logger;
```

Comment:
When we are not using jobLogger, I suppose this is not necessary anymore. Instead use "this.logger" everywhere.